app/uk/gov/openregister/config/FieldRegister.java

```diff
@@ -14,7 +15,7 @@
 
 public class FieldRegister extends Register {
 
-    public static final List<Field> FIELDS = Arrays.asList(new Field("field"), new Field("datatype"), new Field("register", Optional.of("register")), new Field("cardinality"), new Field("text"));
+    public static final List<Field> FIELDS = Arrays.asList(new Field("field"), new Field("datatype", Datatype.TEXT), new Field("register", Optional.of("register")), new Field("cardinality"), new Field("text"));
```

Comment:
shouldn't you have added `Datatype.TEXT` to the "text" field, not the "datatype" field?